app/services/platform/submitter_payload.rb

```diff
@@ -196,5 +196,13 @@ def inject_reference_payment_content(text)
     def payment_reference
       "#{ENV['PAYMENT_LINK']}#{user_data['moj_forms_reference_number']}"
     end
+
+    def confirmation_email
+      if user_data[ENV['CONFIRMATION_EMAIL_COMPONENT_ID']] == ENV['SERVICE_EMAIL_OUTPUT']
+        user_data[ENV['CONFIRMATION_EMAIL_COMPONENT_ID']].gsub('@', '+confirmation@')
```

Comment:
Will this actually send/be received for outlook etc?

Reply:
Should do, the team have been testing this in test environment over the last couple of days, and have received both emails. Let me ask them which email provider they've tested with so we can be sure which ones have worked.

Reply:
Forgot to update. The team have tested gmail and outlook and they have received the emails.